website/docs/docs/cloud/manage-access/self-service-permissions.md

```diff
@@ -31,6 +31,8 @@ You will need owner and member groups to help with account onboarding, but these
 
 After onboarding administrative users and configuring RBAC/SSO groups, we recommend the following steps for onboarding users to a dbt Cloud Enterprise account.
 
+TODO: maybe these suggestions should change now that there is a specific License type for managing users, groups, licenses, etc.
+
```

Comment:
This is the part I was least sure about, so I left a TODO here. This could use some help from @saimaddali and/or @matthewshaver

Reply:
[Created an issue](https://github.com/dbt-labs/docs.getdbt.com/issues/3520) for this and will address as I work on updating the broader permissions pages.